pegasus/sites.v3/code.org/public/educate/curriculum/courses.haml

```diff
@@ -1,6 +1,5 @@
 ---
 title: Courses
-theme: responsive
```

Comment:
Yes, this is one of our English-only, non-responsive pages.  The course table is not designed to be responsive at this stage.